app/Providers/PosterServiceProvider.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Providers;
+
+use App\Service\PictureService;
+use Illuminate\Filesystem\Filesystem;
+use Illuminate\Support\ServiceProvider;
+use League\Glide\Responses\LaravelResponseFactory;
+use League\Glide\ServerFactory;
+
+class PosterServiceProvider extends ServiceProvider
+{
+    /**
+     * Loading services will be deferred.
+     *
+     * @var bool
+     */
+    protected $defer = true;
+
+    /**
+     * Bootstrap the application services.
+     *
+     * @return void
+     */
+    public function boot()
```

Comment:
Empty method but you can't remove it or you forgot to remove ? 😛

Reply:
artisan command populate the class I forgot to remove it.